bin/start-func-test-server.js

```diff
@@ -0,0 +1,191 @@
+/* @flow */
+/* eslint-disable no-console */
+// This starts a docker server for functional tests to access.
+// The script waits for the server to start and prints some logs
+// to help with debugging.
+
+const fs = require('fs');
+const path = require('path');
+const childProcess = require('child_process');
+
+const root = path.join(__dirname, '..');
+console.log(`Working directory: ${root}`);
+const containerIdFile = path.join(root, 'docker-container-id.txt');
+
+// TODO: make these configurable when we need to start
+// servers for multiple apps.
+const appInstance = 'disco';
```

Comment:
Heh, I thought so too but passing env vars to commands in [.travis.yml](https://github.com/kumar303/addons-frontend/blob/4eea1340c0ec9d3e398c47aded50a2e32367d741/.travis.yml#L30) is not straight forward. I think it's best to deal with it later. I think we may have to create an `amo` and a `disco` `package.json` script.